spec/tracker/tracker_spec.rb

```diff
@@ -12,6 +12,11 @@ class BodyHandler < DummyHandler
   self.position = :body
 end
 
+class BodyOpeningHandler < DummyHandler
+  self.position = :body
+  self.exact_position = :opening
```

Comment:
hm... I mean it's not critical but I don't really like the semantics here. With your change position is not precise anymore. Maybe we should call it container or something, and let the position be opening/closing ?